architecture/core_concepts/pods_and_services.adoc

```diff
@@ -202,6 +209,98 @@ of internal IPs.
 The link:http://kubernetes.io/docs/user-guide/services/[Kubernetes
 documentation] has more information on services.
 
+[[service-externalip]]
+=== Service externalIPs
+
+In addition to the cluster internal IP addresses, the user can configure IP addresses that are external
+to the cluster. The user is responsible for ensuring that traffic arrives at a node with this IP.
+
+The externalIPs must be selected by teh admin from the `*ExternalIPNetworkCIDRs*` range confiugred in 
+xref:../admin_guide/tcp_ingress_external_ports.adoc#unique-external-ips-ingress-traffic-configure-cluster[master.yaml]
+file.
+
+.Sample ExternalIPNetworkCIDR /etc/origin/master/master-config.yaml
+====
+----
+networkConfig:
+  ExternalIPNetworkCIDR: 172.47.0.0/24
+----
+====
+
+.Service externalIPs Definition (JSON)
+====
+
+[source,json]
+----
+{
+    "kind": "Service",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "my-service"
+    },
+    "spec": {
+        "selector": {
+            "app": "MyApp"
+        },
+        "ports": [
+            {
+                "name": "http",
+                "protocol": "TCP",
+                "port": 80,
+                "targetPort": 9376
+            }
+        ],
+        "externalIPs" : [
+            "80.11.12.10"         <1>
+        ]
+    }
+}
+----
+
+<1> List of External IP addresses on which the *port* is exposed. In addition to the internal IP addresses)
+
+====
+
+[[service-ingressip]]
+=== Service ingressIPs
+
+In non-cloud clusters, externalIP addresses can be automatically assigned from a pool of addresses.
+This eliminates the need for the administrator manually assigning them.
+
+The pool is configured in *_/etc/origin/master/master-config.yaml_* file. After changing this file restart 
+the master service.
+
+The `ingressIPNetworkCIDR` is set by default to `172.46.0.0/16`. You could use the default if
+your cluster environment is not already using this private range. However, if you want to use a different range.
```

Comment:
Should also mention that if you make an ipfailover for the range, it needs to be smaller.